src/app/pdf-viewer/pdf-viewer.component.ts

```diff
@@ -183,6 +185,11 @@ export class PdfViewerComponent
   set fitToPage(value: boolean) {
     this._fitToPage = Boolean(value);
   }
+  
+  @Input('removePageBorders')
```

Comment:
Preferred naming case is kebab-case.
And maybe it's better to have something like `show-borders` instead?